lib/data/building_singleton.dart

```diff
@@ -1,5 +1,6 @@
 import '../models/building.dart';
 import '../models/coordinate.dart';
+import '../models/coordinate.dart';
```

Comment:
I forgot to mention this duplication in my last review

Reply:
Corrected!

Reply:
Seems like you fixed it as I sent my last comment 👍